dawn/src/dawn/IIR/Stage.cpp

```diff
@@ -32,8 +32,8 @@
 namespace dawn {
 namespace iir {
 
-Stage::Stage(const StencilMetaInformation& metaData, int StageID)
-    : metaData_(metaData), StageID_(StageID), iterationSpace_() {}
+Stage::Stage(const StencilMetaInformation& metaData, int StageID, IterationSpace iterationSpace)
```

Comment:
This allows us to specify an IterationSpace without an Interval?

Reply:
The constructor called in the existing deserialize method, passes only metadata and stencil ID, so this allows an iteration space to be passed in there as well.